Feature branch

MarvellousUbani opened the pull request.

samgaco commented on re-former/config/routes.rb:
```diff
@@ -0,0 +1,5 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :users, :only => [:new, :create, :edit] 
```
- [ ] we should have a route for `update` as well

samgaco commented on re-former/app/controllers/users_controller.rb:
```diff
@@ -0,0 +1,27 @@
+	end
+
+	def create
+		  # @user = User.new(username: params[:username], email: params[:email], password: params[:password])
```
- [ ] try to remove commented code in your app

samgaco commented on re-former/app/controllers/users_controller.rb:
```diff
@@ -0,0 +1,27 @@
+
+	def edit
+		@user = User.find(params[:id])
+	end
```
- [ ] we should have an update method as well, so we can update the information of our user when we edit.

samgaco commented on re-former/app/views/users/new.html.erb:
```diff
@@ -0,0 +1,32 @@
+<!-- <form action="/users" method="post" accept-charset="UTF-8" >
+  <label for="username"> Type in username</label> <br> 
```
- [ ] you should have `form_tag` commented as well

> Railsy Forms with #form_tag
> Now we’ll start morphing our form into a full Rails form using the #form_tag and #*_tag helpers. There’s actually very little additional help that’s going on and you’ll find that you’re mostly just renaming HTML tags into Rails tags.

MarvellousUbani commented:
> Hello and Thanks for your feedback.
We've been able to add in the update method on the users controller and also test that it works well. 
We've also removed the specified comment from the code and added in the form_tag method commented out as well.

mosaaleb commented on re-former/app/models/user.rb:
```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+	validates :username, presence:true
```
- We can validate multiple attributes in the same line if they all share the same validator
`validates :username, :email, :password, presence: true`
